scripts/create_test_subset.py

```diff
@@ -111,13 +134,25 @@ def main(
     pid_sid = papi.get_external_participant_id_to_internal_sample_id(project)
     sample_id_by_participant_id = dict(pid_sid)
 
-    if families_n is not None:
+    if families_n:
+        if _additional_samples:
+            _additional_fams.extend(
+                get_fams_for_samples(
+                    project,
+                    _additional_samples,
+                )
+            )
         fams = fapi.get_families(project)
-        all_families = [family['id'] for family in fams]
+        families_to_subset = [
+            family['id']
+            for family in fams
+            if family['external_id'] not in _additional_fams
```

Comment:
Not a big deal for this script with likely small amount of additional family IDs, but if doing existence checks, good to make sure `_additional_families` is a set for O(1) lookups instead of O(n).